examples/service-clients/azure-client/external-controller/src/app.ts

```diff
@@ -167,6 +177,12 @@ async function start(): Promise<void> {
 
 	document.title = id;
 
+	// Biome insist on no semicolon - https://dev.azure.com/fluidframework/internal/_workitems/edit/9083
+	// eslint-disable-next-line @typescript-eslint/member-delimiter-style
+	const lastRoll: { die1?: DieValue; die2?: DieValue } = {};
```

Comment:
Particularly to this example - what is the advantage of using PresenceManager vs SharedMap? Maybe description or even better, ReadMe could talk about that.

Reply:
The example package readme?
Might want to have more of that on the real examples (other repos), here the example isn't meant to be about presence and doesn't demonstrate things we really are promoting. (SharedMap is not something we want to point people too either.)
Unfortunately, there are few examples that use the public API. So this one was what I could reasonably change without reinventing.

SharedMap (or SharedTree) are quite different. The Presence package readme does not explicitly compare. I feel like it stands on it own and people will get the difference. But if you've read https://github.com/microsoft/FluidFramework/blob/main/packages/framework/presence/README.md and https://fluidframework.com/docs/data-structures/overview/ and think we should do more, sure.